File: JXXZ.ZHCG/JXXZ.ZHCG.WebAPI/app/model/sys/Permission.js
Ext.define('TianZun.model.sys.Permission', {
    extend: 'Ext.data.Model',
    fields: [
        { name: 'Code', type: 'string' },
        { name: 'Name', type: 'string' },
        { name: 'ParentCode', type: 'string' },
        { name: 'Path', type: 'string' },
        { name: 'Comment', type: 'string' },
        { name: 'SeqNo', type: 'int' },
        //{
        //    name: 'checked',
        //    convert: function (value, record) {
        //        if (value == undefined || value == null) {
        //            return false;
        //        } else {
        //            return value;
        //        }
        //    }
        //}
    ]
});

File: JXXZ.ZHCG/JXXZ.ZHCG.Model/Permission.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace JXXZ.ZHCG.Model
{
    public class Permission
    {
        public string Code { get; set; }
        public string Name { get; set; }
        public string ParentCode { get; set; }
        public string Path { get; set; }
        public string Comment { get; set; }
        public int? SeqNo { get; set; }
    }
}
